site/docs/about/guidebook1.0.mdx

```diff
@@ -0,0 +1,55 @@
+---
+name: Guidebook 1.0
+route: /about/guidebook1.0
+menu: About
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Link from "../../src/components/Link";
+
+# HDS guide for start using version 1.0
+
+<LargeParagraph>
+    Here you will find some useful information to make your transition to HDS 1.0 as smooth as possible.
+</LargeParagraph>
+
```

Comment:
Should we add a "TLDR" section just below the lead paragraph? Just to list breaking changes and provide link to the chapter detailing them. Most people won't read this thoroughly anyway so providing a quick list of things they need to note could be good.

Reply:
Sounds good